### ISSUE TITLE
Unable to start cluster after update

### ISSUE THREAD

**Issue by Tiffany-silva** [wso2/product-is#19990]
**Describe the issue:**
After updating to the latest U2 level the server couldn't start the cluster due to a ClassNotFoundException error:
```
WARN {com.hazelcast.spi.EventService} - [,]:[10.209.3.132]:5701 [cbsa.is.domain] [3.12.13] Error while logging processing event com.hazelcast.nio.serialization.HazelcastSerializationException: Problem while reading Externalizable class: org.wso2.carbon.core.clustering.hazelcast.IdempotentWrappedClusteringMessage, exception: java.lang.ClassNotFoundException: org.wso2.carbon.core.clustering.hazelcast.IdempotentWrappedClusteringMessage cannot be found by hazelcast_3.12.2.wso2v1
        at com.hazelcast.internal.serialization.impl.JavaDefaultSerializers$ExternalizableSerializer.read(JavaDefaultSerializers.java:159)
        at com.hazelcast.internal.serialization.impl.JavaDefaultSerializers$ExternalizableSerializer.read(JavaDefaultSerializers.java:135)
        at com.hazelcast.internal.serialization.impl.StreamSerializerAdapter.read(StreamSerializerAdapter.java:48)
        at com.hazelcast.internal.serialization.impl.AbstractSerializationService.toObject(AbstractSerializationService.java:187)
        at com.hazelcast.topic.impl.DataAwareMessage.getMessageObject(DataAwareMessage.java:44)
        at org.wso2.carbon.hazelcast.HazelcastIdempotentClusterMessageListener.onMessage(HazelcastIdempotentClusterMessageListener.java:49)
        at com.hazelcast.topic.impl.TopicService.dispatchEvent(TopicService.java:138)
        at com.hazelcast.spi.impl.eventservice.impl.EventProcessor.process(EventProcessor.java:61)
        at com.hazelcast.spi.impl.eventservice.impl.RemoteEventProcessor.run(RemoteEventProcessor.java:47)
        at com.hazelcast.util.executor.StripedExecutor$Worker.process(StripedExecutor.java:244)
        at com.hazelcast.util.executor.StripedExecutor$Worker.run(StripedExecutor.java:227)
```
Class `org.wso2.carbon.core.clustering.hazelcast.IdempotentWrappedClusteringMessage` was renamed to `org.wso2.carbon.hazelcast.IdempotentWrappedClusteringMessage` according to [1] but it was not updated accordingly on the `hazelcast_3.12.2.wso2v1` component.

[1] https://github.com/wso2/product-is/pull/18809/

**How to reproduce:**
Update from U2 level 216 (or below) to U2 level 271 (or above)

**Expected behavior:**
Clusters should start seamlessly after taking updates.

**Environment information:**
 - Product Version: IS 5.10.0

**Comment by DInuwan97** [comment]
Hi @RushanNanayakkara / @Tiffany-silva 

This setup involves WSO2 Identity Server KM 5.10.0. However, it cannot be reproduced exactly at the same U2 level. As per [1], the following changes were implemented [2]. In these changes, the package name was altered from org.wso2.carbon.core.clustering.hazelcast to org.wso2.carbon.hazelcast. Furthermore, the clustering membership scheme is unrelated, as Hazelcast is common across all clustering membership schemes. Because `HazelcastClusteringAgent` imports all the clustering schemas like AWS,WKA,Kubernetes etc. I was able to reproduce the scenario using two different U2 levels with the WKA membership scheme, while maintaining the following Hazelcast configurations.

```
[hazelcast]
hazelcast.shutdownhook.enabled = "false"
hazelcast.logging.type= "log4j2"
hazelcast.heartbeat.interval.seconds = 1
hazelcast.master.confirmation.interval.seconds = 5
hazelcast.max.no.heartbeat.seconds = 20
hazelcast.max.no.master.confirmation.seconds = 30
hazelcast.version.check.enabled = "false"
hazelcast.phone.home.enabled = "false"

[clustering]
membership_scheme = "wka"
local_member_host = "127.0.0.1"
local_member_port = "4001"
members = ["[127.0.0.1:4000](http://127.0.0.1:4000/)", "[127.0.0.1:4001](http://127.0.0.1:4001/)"]
```

However it is a must requirement to update the both nodes to same U2 level specially in a clustering environment.

[1] - https://github.com/wso2/product-is/pull/18809
[2] - https://github.com/wso2/carbon-kernel/pull/3697/files

Thanks
